docs/user-guide/cli-using-initializing-user-configuration.md

```diff
@@ -0,0 +1,70 @@
+# Initializing user configuration
```

Comment:
@adam-wolfe @zFernand0: 

Based on a discussion w/ Fernando and @gejohnston, I moved this article farther down on the left-side Table of Contents. I also rewrote the introduction to emphasize that user configs are often not used. Also rewrote Step 1 in the **Creating user profiles** section. 
Please review. Let me know if this wasn't the right approach.

Reply:
Looks good to me.